test/indexer/test_indexer.py

```diff
@@ -192,12 +197,13 @@ def test_deletion(self):
                     self.assertEqual(len(hits), 2 * size)
                     docs_by_entity: dict[EntityReference, list[Contribution]] = defaultdict(list)
                     for hit in hits:
+                        # FIXME: Why parse the hit as a contribution before asserting doc type?
```

Comment:
Generally FIXME comments are added with a GitHub ticket URL. Did you intend to leave this comment as is or return to this for more changes in this PR?

Reply:
Resolved at PL. Comment removed.